.github/workflows/ci.yml

```diff
@@ -80,7 +81,7 @@ jobs:
 
   telegram:
     needs: [pre_job, database_lint, database_tests]
-    if: ${{ needs.pre_job.outputs.should_skip != 'true' }}
+    if: ${{ always() && needs.pre_job.outputs.should_skip != 'true' }}
```

Comment:
how does this make failed runs also get sent

Reply:
- there is an inherent if statement to check that the previous tests have passed
- always() kind of "ignores" that if statement